crypto/merklearray/proof_test.go

```diff
@@ -157,7 +157,7 @@ func TestConcatenatedProofsMissingChild(t *testing.T) {
 	err = Verify(tree.Root(), map[uint64]crypto.Hashable{6: array[6]}, newP.ToProof())
 	a.NoError(err)
 
-	recomputedProof, err := ProofDataToSingleLeafProof(p.HashFactory.HashType.String(), uint64(p.TreeDepth), concatenatedProof)
+	recomputedProof, err := ProofDataToSingleLeafProof(p.HashFactory.HashType.String(), concatenatedProof)
 	a.NoError(err)
```

Comment:
It looks like you can turn these into test cases:
```suggestion
	a.NoError(err)
	a.Equals(p.TreeDepth, recomputedProof.TreeDepth)
```

Reply:
> It looks like you can turn these into test cases:

Good idea -- thanks!